terraform/modules/app-ecs-services/templates/alertmanager.tpl

```diff
@@ -62,7 +62,7 @@ route:
         severity: constant
       receiver: "dev-null"
     - match:
```

Comment:
match will need to be match_re I think to make use of regex

Reply:
👍 ?